janusgraph-hbase-parent/janusgraph-hbase-core/src/test/java/org/janusgraph/blueprints/HBaseGraphProvider.java

```diff
@@ -28,16 +26,11 @@
 
     @Override
     public ModifiableConfiguration getJanusGraphConfiguration(String graphName, Class<?> test, String testMethodName) {
-        return HBaseStorageSetup.getHBaseConfiguration(graphName);
-    }
-
-    @Override
-    public Graph openTestGraph(final Configuration config) {
         try {
             HBaseStorageSetup.startHBase();
         } catch (IOException e) {
             throw new RuntimeException(e);
         }
-        return super.openTestGraph(config);
+        return HBaseStorageSetup.getHBaseConfiguration(graphName);
```

Comment:
(nitpick) Are you able to shutdown? Hbase correctly.

Reply:
I don't know how I should do that if the JUnit annotations don't work. But we are doing it the same way for [CQL](https://github.com/JanusGraph/janusgraph/blob/master/janusgraph-cql/src/test/java/org/janusgraph/blueprints/cql/CQLGraphProvider.java).
Wouldn't the process be stopped automatically when the test finishes?

Reply:
It's okay to leave it as it is.